modules/spo-binding-workloads.adoc

```diff
@@ -35,10 +35,11 @@ spec:
   profileRef:
     kind: {kind} <1>
     name: profile <2>
-  image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21
```

Comment:
What do you think if we keep this one as original, but still point this line to #3 and saying "*" string to bind the profile to all pods in the bottom #3 details

Also noted that they need to use quote around the * like "*" in the image filed